src/main/java/uk/gov/hmcts/ccd/data/user/JurisdictionsResolver.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.ccd.data.user;
+
+import java.util.List;
+
+public interface JurisdictionsResolver {
+
+    List<String> getJurisdictionsFromIdam(String[] roles);
```

Comment:
this method interface signature should be more generic. The fact that jurisdictions are retrieved from Idam is a detail of one of the interface implementations. Other implementation might retrieve jurisdiction not from Idam but from other places for example user profile. Same for the parameter. It's specific of the implementation and should not appear here. We should rename to:
`getJurisdictions()`

Reply:
addressed